CIP-Improved_Rewards_Scheme_Parameters/README.md

```diff
@@ -0,0 +1,237 @@
+---
+CIP: 82
+Title: Improved Rewards Scheme Parameters
+Status: Project Area Not Enlisted for CIP Process
```

Comment:
```suggestion
Status: Inactive (project area not enlisted for CIP process)
```
This is a qualifying phrase for the status of `Inactive`, rather than the status itself... lowercase according to current example in [CIP-0001 > Status: Inactive](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#status-inactive).